### DISCUSSION TITLE
GyroDSU over Remote Play

### DISCUSSION THREAD

**Discussion by FrostyBits77** [kmicki/SteamDeckGyroDSU#52] [Ideas]
I am streaming Tears of The Kingdom from my pc to Steam Deck. Please add a feature for my pc to pick up my SD's GyroDSU over remote play. Steam Remote Play/Moonlight, either or both would be great. Hoping it's possible and easy to do.

**Comment by kmicki** [comment]
It should be already possible.
In Controls configuration click "Configure" under "Motion", remove default server and add new server with IP of Steam Deck.

![image](https://github.com/kmicki/SteamDeckGyroDSU/assets/1463619/9698020b-71a7-4441-be3c-1491349bd7a1)

**Reply by FrostyBits77** [reply]
This worked, thank you so much.
